lib/takeaway.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'menu.rb'
+require_relative 'order.rb'
+require 'dotenv'
+require 'rubygems'
+require 'twilio-ruby'
+
+class TakeAway
+  
+  def initialize(order = Order.new)
+    @order = order
+    @env = Dotenv.load
+  end
+  
+  def read_menu
+    @order.menu
+  end
+  
+  def complete_order(price)
+    fail "Value does not match the total" unless correct_amount?(price)
+    send_text("Thank you! Your order was placed.")
+  end
+  
+  def order(dish, quantity = 1)
+    add_dish_to_order(dish, quantity)
+  end
+  
+  def basket_summary
+    @order.current_order
+  end
+  
+  def total
+    "Total: £" + "%.2f" % @order.total_amount
+  end
+  
+  private 
+  
+  def add_dish_to_order(dish, quantity)
+    @order.add_dish_to_basket(dish, quantity)
+  end
+  
+  def send_text(message)
```

Comment:
You should add a new class for messager. See the link below: 
https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#design-for-single-responsibility-principle